src/features/game/expansion/placeable/Placeable.tsx

```diff
@@ -225,10 +227,7 @@ export const Placeable: React.FC = () => {
                 height: `${dimensions.height * GRID_WIDTH_PX}px`,
               }}
             >
-              {PLACEABLES[placeable]({
-                coordinates,
-                grid,
-              })}
+              <Collectible grid={grid} coordinates={coordinates} />
```

Comment:
This prevents the following error:

`Uncaught Error: Rendered fewer hooks than expected. This may be caused by an accidental early return statement in React Hooks)`